src/Framework/TestSuite.php

```diff
@@ -404,14 +420,19 @@ public function addTestFiles($filenames)
     /**
      * Counts the number of test cases that will be run by this test.
      *
+     * @Param boolean $preferCache Indicates if cache is preferred.
      * @return integer
      */
-    public function count()
+    public function count($preferCache = false)
```

Comment:
Ok. I have moved this to https://github.com/sebastianbergmann/phpunit/pull/1627.